roles/splunk/tasks/kvstore_upgrade.yml

```diff
@@ -0,0 +1,152 @@
+---
+- name: Set fact for kvstore Upgrade
+  ansible.builtin.set_fact:
+    splunk_upgrade_server: false
+
+- name: Check the current kvstore storage backend
+  ansible.builtin.command: |
+    {{ splunk_home }}/bin/splunk show kvstore-status --verbose | grep storageEngine | sed -r 's/\s+storageEngine : //g'
```

Comment:
Works without `--verbose` too

Reply:
Docs say to use `--verbose` so I'm going to keep it that way: https://docs.splunk.com/Documentation/Splunk/latest/Admin/MigrateKVstore